tests/repo_finder/test_repo_finder.py

```diff
@@ -74,34 +75,49 @@ def test_resolve_analysis_target(
     assert Analyzer.to_analysis_target(config, available_domains) == expect
 
 
-def test_pom_extraction_ordering() -> None:
+@pytest.mark.parametrize(
+    ("user_config_input", "expected"),
+    [
+        (
+            """
+                [repofinder.java]
+                repo_pom_paths =
+                    scm.connection
+                    scm.url
+                """,
```

Comment:
Hmm is the extra spaces before `"""` here intentional? Otherwise, everything else looks good to me.